.travis.yml

```diff
@@ -0,0 +1,9 @@
+sudo: false
+language: node_js
+node_js:
+  - "6"
+install:
+  - make deps
+script:
+  - make test
```

Comment:
I get one test failing:
  1) Event hub functions unit tests. Simple OK test, activity record:
     ReferenceError: ehubActivityLogsWire is not defined
      at Context.<anonymous> (test/ehub_collector_test.js:4:29)

Reply:
Plus there are multiple warnings returned by jshint.

Reply:
OTOH, I guess this isn't really about adding travis-ci, so maybe that should be handled outside of this review.

Reply:
Warning will be fixed in a separate PR

Reply:
ah, sorry, the first comment is my bad - I had a local file where I was testing something